masterlist.yaml

```diff
@@ -1796,9 +1796,12 @@ plugins:
       # - Text
     dirty:
       - <<: *quickClean
-        crc: 0xCB6B7C8A
-        util: '[FO4Edit v4.1.5k](https://www.nexusmods.com/fallout4/mods/2737)'
+        crc: 0x9DD10683
+        util: '[FO4Edit v4.1.5f](https://www.nexusmods.com/fallout4/mods/2737)'
         itm: 7
+    clean:
+      - crc: 0x9A5A921E
+        util: 'FO4Edit v4.1.5f'
```

Comment:
I'm assuming that's the CRC after cleaning.
It's fine to add them, but it's only useful if using that version of xEdit.